app/(routes)/page.tsx

```diff
@@ -11,16 +11,21 @@ import {
 } from "@components/_shared";
 import { ProposalsListComponent } from "@components/proposals-list/proposals-list.component";
 import { Loader } from "@components/_shared";
-import NumbersService from "@/app/helpers/numbers.service";
 import { ContractParams } from "@components/contract-params/contract-params.component";
 import { useRouter } from "next/navigation";
 import { useUserStore } from "@/app/store";
 import { Suspense } from "react";
 import ProposalSummaryComponent from "@components/proposal-summary/proposal-summary.component";
+import { useAccount } from "wagmi";
 
 const Page = () => {
   const router = useRouter();
   const { walletAddress } = useUserStore();
+  const { chain } = useAccount();
+
+  const chainKind = chain?.testnet ? "Testnet" : "Mainnet";
+  const chainText = chain?.name || "CELO";
```

Comment:
Could use useMemo here as the useAccount hook might trigger refresh
```suggestion
  const chainText = useMemo(
    () => (chain?.name || "CELO"),
    [chain?.name],
  );
```